project_template/package.json

```diff
@@ -53,38 +52,39 @@
     "eslint-plugin-compat": "^1.0.4",
     "eslint-plugin-prettier": "^2.1.2",
     "eslint-plugin-react": "^7.1.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "file-loader": "^0.11.2",
-    "html-webpack-plugin": "^2.28.0",
+    "extract-text-webpack-plugin": "4.0.0-beta.0",
```

Comment:
we need to keep an eye on these beta dependencies

Reply:
Yep. I was hoping these plugins would be updated to final by the time we merge this.